dags/pubmed.py

```diff
@@ -22,15 +22,22 @@
     command='make start pubmed 1900-01-01 2100-01-01'
 )
 
-processor_task = helpers.create_processor_task(
-    name='pubmed',
+unregistred_trials_task = helpers.create_processor_task(
+    name='pubmed_unregistered_trials',
     dag=dag
 )
 
-pubmed_linker_task = helpers.create_processor_task(
-    name='pubmed_linker',
+trials_remover_task = helpers.create_processor_task(
+    name='trial_remover',
     dag=dag
 )
 
-processor_task.set_upstream(collector_task)
-pubmed_linker_task.set_upstream(processor_task)
+publications_task = helpers.create_processor_task(
+    name='pubmed_publications',
+    dag=dag
+)
+
+
+unregistred_trials_task.set_upstream(collector_task)
+trials_remover_task.set_upstream(unregistred_trials_task)
+publications_task.set_upstream(trials_remover_task)
```

Comment:
The order `collector -> unregistered_trials -> trials_remover -> publications_task` feels odd. Why the `publication_task` depends on the `unregistered_trials` or the `trials_remover`? It feels like both should depend only on the `collector`, and the `publication_task` depend on both of them. What am I missing?

There's also a small typo (`s/unregistred/unregistered`) and please rename `publications_task` to `pubmed_publications_task` to keep the pattern.

Other than that, all good :+1: